trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './nameable'
```

Comment:
- [ ] @LeslieAine kindly ensure you are importing the correct file since you are inheriting from the base decorator it should be base_decorator and not nameable, also ensure you can manage to decorate your person using the example given 
![image](https://user-images.githubusercontent.com/61408860/233104251-1c344ce7-7a50-4ae8-aa99-f34290402f89.png)

Reply:
Thank you Dan